src/ui/zcl_abapgit_gui_page_repo_over.clas.abap

```diff
@@ -48,102 +49,120 @@ CLASS zcl_abapgit_gui_page_repo_over DEFINITION
         deserialized_by TYPE xubname,
         deserialized_at TYPE string,
         write_protected TYPE abap_bool,
-      END OF ty_overview,
+      END OF ty_overview .
+    TYPES:
       ty_overviews TYPE STANDARD TABLE OF ty_overview
-                   WITH NON-UNIQUE DEFAULT KEY.
+                     WITH NON-UNIQUE DEFAULT KEY .
+
     CONSTANTS:
       BEGIN OF c_action,
         select       TYPE string VALUE 'select',
         apply_filter TYPE string VALUE 'apply_filter',
       END OF c_action .
+    DATA mv_order_descending TYPE abap_bool .
+    DATA mv_filter TYPE string .
+    DATA mv_time_zone TYPE timezone .
+    DATA mt_col_spec TYPE zif_abapgit_definitions=>ty_col_spec_tt .
+    DATA mt_overview TYPE ty_overviews .
```

Comment:
suggest rolling back this file, and only staging the changed lines. This PR is 1000+ lines which already makes it difficult to review. If the change was half the size, I'd be okay with these whitespace changes